scripts/crowdin/pretranslate.mjs

```diff
@@ -0,0 +1,159 @@
+const projectId = process.env.CROWDIN_PROJECT_ID;
+const token =  process.env.CROWDIN_PERSONAL_TOKEN;
+
+const BASE_URL = `https://api.crowdin.com/api/v2/projects/${projectId}`;
+const preTranslateURL = `${BASE_URL}/pre-translations`;
```

Comment:
Uppercase please. it's constant, not a normal variable.

Reply:
same comment applies to `token` and `projectId` as well.

Reply:
Fixed